spec/requests/api/eform_spec.rb

```diff
@@ -108,17 +209,26 @@
     end
 
     it 'responds with 401 if authenticating via token with OBO token' do
-      post '/api/eform/validate', headers: { 'Authorization' => 'Bearer ' + api_token_with_obo_token_from_response(file_fixture('oam/sso_response_success.xml').read) }, params: { identifier: 'App.GeneralAgenda', version: '1.7', data: general_agenda }
+      post '/api/eform/validate', headers: { 'Authorization' => 'Bearer ' + api_token_with_obo_token_from_response(file_fixture('oam/sso_response_success.xml').read) }, params: { identifier: 'App.GeneralAgenda', version: '1.9', data: general_agenda }
 
       expect(response.status).to eq(401)
       expect(response.body).to eq({ message: 'Bad credentials' }.to_json)
     end
 
     it 'responds with 404 if request contains form which can not be found' do
-      post '/api/eform/validate', headers: { 'Authorization' => 'Bearer ' + token }, params: { identifier: 'App.GeneralAgenda', version: '1.9', data: general_agenda }
+      post '/api/eform/validate', headers: { 'Authorization' => 'Bearer ' + token }, params: { identifier: 'App.UnknownAgenda', version: '1.0', data: general_agenda }
 
       expect(response.status).to eq(404)
-      expect(response.body).to eq({ message: 'Form App.GeneralAgenda version 1.9 not found' }.to_json)
+      expect(response.body).to eq({ message: 'Form App.UnknownAgenda version 1.0 not found' }.to_json)
+    end
+
+    it 'responds with 409 if request contains form which has no schema' do
```

Comment:
Imho to je 404 https://httpstatuses.com/404 a asi by sme tam mali poslat vnutri ten kod co posiela upvs

Reply:
na 404 mi to nesedi, lebo form template ako taky existuje, len nema XSD schemu. mohlo by to byt este nieco ine? co to odlisi od 404